src/javascript/JContent/ContentRoute/ContentLayout/EmptyDropZone/EmptyDropZone.jsx

```diff
@@ -46,18 +60,18 @@ const EmptyDropZone = ({component: Component, t, classes, mode}) => {
         );
     }
 
-    if (mode === JContentConstants.mode.PAGES) {
+    if (mode === JContentConstants.mode.CONTENT_FOLDERS) {
```

Comment:
Do we need to have permissions check here as well or no?

Reply:
We do as for CONTENT_FOLDERS we use import instead of upload as text
Will add that permission in my next PR